css/style.css

```diff
@@ -0,0 +1,135 @@
+body {
+    font-size: 18px;
+    font-family: "Open Sans", Arial, sans-serif;
+
+}
+
+
+.wrapper { 
+    max-width: 1600px;
+    margin: 0 auto;
+}
+
+.social_link {
+    position: relative;
+    top: 30px;
+    right: 215px;
+    color: #FFFFFF;
+    float: right;
+    margin-right: 15px;
+}
+
+.social a:hover{
+    color: #03204f;
+    border-radius: 2px;
+}
+
+.banner {
+    padding: 174px 215px;
+    background: url("../img/banner.png") no-repeat center top;
+}
+
+.banner_info h1,h2 {
+     width: 800px;
+     height: 55px;
+     font-family: Open Sans;
+     font-style: normal;
+     color: #fff;
+     font-weight: normal;
+     line-height: 66px;
+     font-size: 44px;
+     letter-spacing: 0.016em;
+    }
+
+    .btn {
+        font-family: "Open Sans";
+        font-style: normal;
+        font-weight: normal;
+        line-height: normal;
+        font-size: 18px;
+        text-align: center;
+        letter-spacing: 0.016em;
+        text-transform: capitalize;
+        display:inline-block;
+        padding: 24x;
+        background: #FFFFFF;
+        color: #2F80ED;
+        border-radius: 144px;
+        cursor: pointer;
+        width: 282px;
+        height: 68px;
+        border: none;
+    }
+    .btn:hover {
+        background-color: #b0caf2;
+        color: #FFFFFF;
+    }
+
+    .heading {
+        padding: 80px 218px;
+    }
+
+    .heading p {
+        line-height: 66px;
+        font-family: "Open Sans";
+        font-style: normal;
```

Comment:
font-style: normal; and font-weight: normal; is default for the most of the cases, try to review and remove them if possible